diff --git a/scripts/base16-google-dark.sh b/scripts/base16-google-dark.sh
index 5553895..c4985a8 100755
--- a/scripts/base16-google-dark.sh
+++ b/scripts/base16-google-dark.sh
@@ -9,7 +9,7 @@ color02="19/88/44" # Base 0B - Green
 color03="FB/A9/22" # Base 0A - Yellow
 color04="39/71/ED" # Base 0D - Blue
 color05="A3/6A/C7" # Base 0E - Magenta
-color06="39/71/ED" # Base 0C - Cyan
+color06="00/BC/D4" # Base 0C - Cyan
 color07="c5/c8/c6" # Base 05 - White
 color08="96/98/96" # Base 03 - Bright Black
 color09=$color01 # Base 08 - Bright Red
diff --git a/scripts/base16-google-light.sh b/scripts/base16-google-light.sh
index 65a5ca2..cd31280 100755
--- a/scripts/base16-google-light.sh
+++ b/scripts/base16-google-light.sh
@@ -9,7 +9,7 @@ color02="19/88/44" # Base 0B - Green
 color03="FB/A9/22" # Base 0A - Yellow
 color04="39/71/ED" # Base 0D - Blue
 color05="A3/6A/C7" # Base 0E - Magenta
-color06="39/71/ED" # Base 0C - Cyan
+color06="00/BC/D4" # Base 0C - Cyan
 color07="37/3b/41" # Base 05 - White
 color08="b4/b7/b4" # Base 03 - Bright Black
 color09=$color01 # Base 08 - Bright Red
